packages/tokens/fungible/src/utils/sac_admin/README.md

```diff
@@ -0,0 +1,41 @@
+```mermaid
+sequenceDiagram
+    actor Issuer
+    actor Owner
+    participant Admin
+    participant AdminSigner
```

Comment:
I think a few explanations on top of this mermaid diagram would be useful. For example:
- `Admin` is the wrapper contract
- `Owner` can be renamed (may be confusing due to we have ownable etc.)
- an intro like: "this contract is a wrapper contract that can be deployed on top of an existing contract to delegate admin related functions, etc.